src/server/utils.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Breaking out these into a util file is probably a good move, although it's of only marginal utility right now since the code is only being used in `query`.